data_analysis/static_gtfs_analysis.py

```diff
@@ -359,16 +359,15 @@ def download_zip(version_id: str) -> zipfile.ZipFile:
         zipfile.ZipFile: A zipfile for the CTA version id.
     """
     logger.info('Downloading CTA data')
-    CTA_GTFS = zipfile.ZipFile(
-        BytesIO(
+    zipfile_bytes_io = BytesIO(
             requests.get(
                 f"https://transitfeeds.com/p/chicago-transit-authority"
                 f"/165/{version_id}/download"
             ).content
         )
-    )
+    CTA_GTFS = zipfile.ZipFile(zipfile_bytes_io)
     logging.info('Download complete')
-    return CTA_GTFS
+    return CTA_GTFS, zipfile_bytes_io
```

Comment:
Based on this change, I think it's true that it's necessary that we have the BytesIO type data in order to do what we want with the date ranges, however, other parts of the code expect that we have the ZipFile type data. Is it then true that we are essentially returning duplicate data here?
I'm not suggesting that we change this, just want to make sure I'm understanding it correctly.

Reply:
Yeah I didn't have a good way of obtaining the `BytesIO` object that is needed for uploading files to s3. For other parts of the code that expect the `ZipFile` output, I would use something like `CTA_GTFS, _ = download_cta_zip()`, so that the `BytesIO` object isn't created where it isn't needed. 

Maybe creating a different data type would be better?